cmd/controller-manager/app/server.go

```diff
@@ -110,28 +119,31 @@ func Run(opt *options.ServerOption) error {
 	return fmt.Errorf("lost lease")
 }
 
-func startControllers(config *rest.Config, opt *options.ServerOption) func(ctx context.Context) {
-	controllerOpt := &framework.ControllerOption{}
+func startControllers(ctx context.Context, controllerOpt *framework.ControllerOption) {
+	framework.ForeachController(func(c framework.Controller) {
+		if err := c.Initialize(controllerOpt); err != nil {
```

Comment:
I'm curious about this error handling. Is it acceptable for failling to initiate some of controllers (I know it won't go into errors regularly and the logic here is same as it used to be)?